.gitignore

```diff
@@ -1 +1,2 @@
 /release-controller
+.idea/*
```

Comment:
Seems unrelated to Go modules?  I expect this falls under "Patterns which a user wants Git to ignore in all situations" in [`gitignore(1)`][1], which suggests putting these in a file that defaults to `$XDG_CONFIG_HOME/git/ignore`.

[1]: https://git-scm.com/docs/gitignore#_description